cli/config/configuration-qemu-virtiofs.toml.in

```diff
@@ -466,3 +466,17 @@ sandbox_cgroup_only=@DEFSANDBOXCGROUPONLY@
 # Supported experimental features:
 # (default: [])
 experimental=@DEFAULTEXPFEATURES@
+
+# Use VFIO driver for VFIO devices in the VM, as well as the host.
+#
+# When enabled, VFIO devices passed to the container appear as VFIO
+# devices within the container, similar behaviour to a conventional
+# container.  NOTE: The VFIO device within the container will have a
+# different name from the device on the host, since the VM has its own
+# IOMMU/VFIO group numbering.
+#
+# When disabled, the VM will use its native drivers for the VFIO
```

Comment:
I wrote elsewhere that I was expecting this to be per-device and not per-containers.

I see based on this comment that you really mean that as a sort of "backward compatibility" fallback. In that case, it makes more sense. That being said, I don't really understand how the container would use the VFIO device without having a proper name for it.

Without that flag, is there any way to get something sensible in the guest at all?

Reply:
Right to use a VFIO device in the old behaviour the container payload would have to be expecting it explicitly.  It might also need other hacks like mapping the VM's /dev into the container.

Reply:
How about `use_vfio_in_guest` or `use_vfio_driver_in_guest` since we tend to use a verb based config for turning a particular feature on/off ?